styles/Global.css

```diff
@@ -260,23 +264,114 @@ a:hover {
 	min-width: 90vw;
 	min-height: 8vh;
 	border: solid #2c2e33;
-	font-family: mulish;
+	font-family: Mulish;
 	font-size: 20px;
 	text-indent: 40px;
 }
 .fourzerofour--wrapper {
-	background: url(../public/blogvector.svg),
-		linear-gradient(#0d0f16 60%, #331b25 136%);
+	background: url(../public/blogvector.svg), #0d0f16;
 	background-repeat: no-repeat;
 	background-position: bottom;
-	min-height: calc(100vh - 170px);
+	min-height: calc(100vh + 20px);
+}
+.fourzerofour--title-wrapper {
+	position: relative;
+}
+.fourzerofour--title {
+	font-size: 36px;
+	font-family: Raleway;
+	font-weight: 900;
+	text-transform: uppercase;
+	color: white;
+	margin: 0;
+	padding-left: 15px;
+}
+.fourzerofour--title::after {
+	content: 'OOPS!';
+	opacity: 0.05;
+	font-size: 70px;
+	position: absolute;
+	left: -10px;
+	top: -40px;
 }
 .fourzerofour--error {
 	background: url(../public/404triangle.svg);
 	background-repeat: no-repeat;
 	display: flex;
 	flex-direction: column;
 	align-items: center;
+	background-position: center;
+	margin: -60px 15px 0 15px;
+}
+.fourzerofour--h2 {
+	color: #ff455b;
+	font-size: 80px;
+	margin: 0;
+	padding-top: 85px;
+}
+.fourzerofour--h3 {
+	color: #989cac;
+	font-size: 20px;
+	font-family: Mulish;
+	font-style: normal;
+	font-weight: normal;
+	line-height: 1.6;
+	transform: translateY(-55px);
+}
+.breadcrumbs--ul {
+	list-style: none;
+	padding: 0;
+	display: flex;
+	justify-content: flex-end;
+}
+.breadcrumbs--li__with-hyphen,
+.breadcrumbs--li {
+	font-size: 16px;
+	font-family: Mulish;
+	font-style: normal;
+	font-weight: normal;
+	line-height: 1.6;
+	margin: 0 20px;
+	display: flex;
+	justify-content: flex-end;
+	color: #989cac;
+}
+.breadcrumbs--li__with-hyphen::after {
+	content: '─';
+	opacity: 0.5;
+	position: absolute;
+	left: calc(100vw - 85px);
```

Comment:
Just putting this out there `right: 85px;`. Then there's a general rule that you always have at least two anchors. Top/left, bottom/right, ect. This isn't wrong. If you're having trouble positioning the hyphen remember to try setting the parent to `position: relative;`.